File: app/src/main/java/com/statuses/statussavers/HowToUse.java
package com.statuses.statussavers;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.MenuItem;
import android.view.View;

import com.applovin.mediation.MaxAd;
import com.applovin.mediation.MaxAdViewAdListener;
import com.applovin.mediation.MaxError;
import com.applovin.mediation.ads.MaxAdView;


public class HowToUse extends AppCompatActivity {

    MaxAdView maxAdView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_how_to_use_2);
        if(getSupportActionBar()!=null) {
            getSupportActionBar().setTitle("How To Use");
            getSupportActionBar().setDisplayHomeAsUpEnabled(true);
        }
        if(MainActivity.maxAdxInitialised) {
            loadAppLovinAd();
        }
    }

    @Override
    public boolean onOptionsItemSelected(@NonNull MenuItem item) {
        int id = item.getItemId();
        if (id == android.R.id.home)
        {
            finish();
            return true;
        }
        else
        {
            return super.onOptionsItemSelected(item);
        }
    }

    private void loadAppLovinAd() {
        maxAdView = (MaxAdView) findViewById(R.id.maxAd);
        maxAdView.setVisibility(View.VISIBLE);
        maxAdView.setListener(new MaxAdViewAdListener() {
            @Override
            public void onAdExpanded(MaxAd maxAd) {

            }

            @Override
            public void onAdCollapsed(MaxAd maxAd) {

            }

            @Override
            public void onAdLoaded(MaxAd maxAd) {

            }

            @Override
            public void onAdDisplayed(MaxAd maxAd) {

            }

            @Override
            public void onAdHidden(MaxAd maxAd) {

            }

            @Override
            public void onAdClicked(MaxAd maxAd) {

            }

            @Override
            public void onAdLoadFailed(String s, MaxError maxError) {
                maxAdView.loadAd();
            }

            @Override
            public void onAdDisplayFailed(MaxAd maxAd, MaxError maxError) {

            }
        });
        maxAdView.loadAd();
    }
}

File: app/src/main/java/com/statuses/statussavers/HowToUse.kt
package com.statuses.statussavers

import android.os.Bundle
import android.view.MenuItem
import android.view.View
import android.view.ViewGroup
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat
import androidx.core.view.updateLayoutParams
import androidx.core.view.updatePadding
import com.applovin.mediation.MaxAd
import com.applovin.mediation.MaxAdViewAdListener
import com.applovin.mediation.MaxError
import com.applovin.mediation.ads.MaxAdView

class HowToUse : AppCompatActivity() {

    private lateinit var maxAdView: MaxAdView
    private lateinit var rootLayout: View
    private lateinit var scrollView: View

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_how_to_use_2)
        rootLayout = findViewById(R.id.rootLayout)
        scrollView = findViewById(R.id.scrollView)
        maxAdView = findViewById(R.id.maxAd)

        // Handle system insets
        ViewCompat.setOnApplyWindowInsetsListener(rootLayout) { _, insets ->
            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())

            // Push content below the status bar
            scrollView.updatePadding(top = 130)

            // Raise AdView above navigation bar
            maxAdView.updateLayoutParams<ViewGroup.MarginLayoutParams> {
                bottomMargin = systemBars.bottom + 10
            }

            insets
        }
        supportActionBar?.let {
            it.title = "How To Use"
            it.setDisplayHomeAsUpEnabled(true)
        }
        if (!HelperClass.adsDisabled && MainActivity.maxAdxInitialised) {
            loadAppLovinAd()
        }
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        return when (item.itemId) {
            android.R.id.home -> {
                finish()
                true
            }
            else -> super.onOptionsItemSelected(item)
        }
    }

    private fun loadAppLovinAd() {
        maxAdView.visibility = View.VISIBLE

        maxAdView.setListener(object : MaxAdViewAdListener {
            override fun onAdExpanded(maxAd: MaxAd) {}

            override fun onAdCollapsed(maxAd: MaxAd) {}

            override fun onAdLoaded(maxAd: MaxAd) {}

            override fun onAdDisplayed(maxAd: MaxAd) {}

            override fun onAdHidden(maxAd: MaxAd) {}

            override fun onAdClicked(maxAd: MaxAd) {}

            override fun onAdLoadFailed(p0: String, p1: MaxError) {
                maxAdView.loadAd()
            }

            override fun onAdDisplayFailed(p0: MaxAd, p1: MaxError) {}
        })

        maxAdView.loadAd()
    }
}
